### PR TITLE
zoom controls on map

### PR DESCRIPTION
**Pull request by yzemaze**
Follow-up to #3489

100 % zoom (default):
image

62 % zoom:
image

### CODE REVIEW

File: assets/app/view/game/map.rb [hunk 1]
```diff
@@ -35,7 +35,8 @@ def render
         @start_pos = [@cols.first, @rows.first]
         @layout = @game.layout
 
-        @scale = SCALE * map_zoom
+        store(:map_zoom, @map_zoom || Lib::Storage['map_zoom'] || 1, skip: true)
```
Comment (tobymao) [inline assets/app/view/game/map.rb:38]: why did you change this? don't store with the storage, it gets hard

Comment (yzemaze) [reply assets/app/view/game/map.rb:38]: Well, I wasn’t sure about this, hence its own commit. I thought it’s better to store once per render than to read 7 times (iirc) from localStorage.

Comment (tobymao) [reply assets/app/view/game/map.rb:38]: well you have to revert it

**Review by tobymao — APPROVED** [review]
